spec/views/about/index.html.erb_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "about/index.html.erb" do
+  let(:admin1) { Fabricate(:admin) }
+  let(:admin2) { Fabricate(:admin) }
+  let(:user) { Fabricate(:user) }
+
+  context "crawler view" do
+    before do
+      def controller.use_crawler_layout?
+        true
+      end
+      admin1
+      admin2
+    end
+    it "renders admin user page links" do
+      @about = About.new(user)
+
+      render
+
+      expect(rendered).to match("/u/#{admin1.username_lower}")
+      expect(rendered).to match(admin2.small_avatar_url)
+    end
+  end
+
+end
```

Comment:
I added a very similar spec already in https://github.com/discourse/discourse/commit/cc6d722de1d5f2c56799f02d9f2e4e7333d247da, we don't need both. We should delete one of them - I don't mind which.